ihmc-perception/src/main/java/us/ihmc/perception/tools/PerceptionMessageTools.java

```diff
@@ -126,6 +131,12 @@ public static void packImageMessageData(BytePointer dataBytePointer, ImageMessag
       }
    }
 
+   public static void packImageMessageData(ImageMessage imageMessage, BytePointer dataPointer)
```

Comment:
I intended to replace the existing `packImageMessageData` method with this new one, but forgot to remove the old one after making sure this one works :( Thank you for reminding me about it!

Anyhow, I think that could be a separate method that uses this method. Having this method specifically for packing the data is handy (you can see my use case in the tests). On a related note, we have a lot of duplicate code that sets all the same `ImageMessage` variables which could be extracted into one or a few methods. I'd like to make that a separate PR though, since the goal of this PR is extracting the decoding of messages. I can focus on more general `ImageMessage` cleanup after this one.